Description: rename msn to msnp and add msn with port 1863
Origin: debian
Author: Alexander Wirt <formorer@debian.org>
Last-Update: 2007-09-01

--- a/firehol.sh
+++ b/firehol.sh
@@ -914,7 +914,10 @@
 # because the mms module is not there:
 # ALL_SHOULD_ALSO_RUN="${ALL_SHOULD_ALSO_RUN} mms"
 
-server_msn_ports="tcp/6891"
+server_msnp_ports="tcp/6891"
+client_msnp_ports="default"
+
+server_msn_ports="tcp/1863 udp/1863"
 client_msn_ports="default"
 
 server_mysql_ports="tcp/3306"
